### ISSUE TITLE
Come up with a better way of excluding packages that can't build on a particular GHC

### ISSUE THREAD

**Issue by michaelpj** [input-output-hk/cardano-haskell-packages#174]
This is very common, and we're likely to be in situations where most packages won't build with that version, e.g. when we do https://github.com/input-output-hk/cardano-haskell-packages/issues/173.

So we want some way of doing this that doesn't require manually listing out every version that doesn't work!

**Comment by coot** [comment]
We could check the `tested-with` cabal meta data.

**Comment by michaelpj** [comment]
Yeah, that could potentially work. Also we can fix that up with metadata revisions, which is nice. I don't know if we have a nice way to read it, though.
